File: week3/lab_py/exercise3.py
import math as m 

radius = float(input('radius = '))
area = m.pi*m.pow(radius,2)
volume = (4/3)*m.pi*pow(radius,3)

print(f'Circle of \"{radius:.2f}\" is: {area:.2f}')
print(f'Sphere of \"{radius:.2f}\" is: {volume:.2f}')

File: week3/lab_java/Exercise3.java
package lab3;

import java.util.Scanner;

public class Exercise3 {
    static Scanner in = new Scanner(System.in);

    public static void main(String[] args) {
        System.out.println("Enter circle radius: ");
        double r = in.nextDouble();

        System.out.printf("Circle area of radius \"%.2f\" is: %.2f\n", r,Math.PI*r*r);
        System.out.printf("Sphere volume of radius \"%.2f\" is %.2f\n", r,(4.0/3.0)*Math.PI*Math.pow(r, 3));
    }
}

